src/DocsParser.ts

```diff
@@ -156,7 +155,7 @@ export class DocsParser {
 
     const allTokens = md.parse(contents, {});
 
-    const baseInfos = await this.parseBaseContainers(filePath, contents, allTokens);
```

Comment:
this was unused

Reply:
Not sure how this can be removed here but not in the method definition?  Can we back this change out to keep this `feat` PR isolated?

Reply:
i backed it out but am also not 100% sure what you meant - i removed it from three places: the impl and two usages (it would not have built otherwise)